source/deployment/cluster.rst

```diff
@@ -1,116 +1,116 @@
-High Availability (E20) - Beta
+High Availability Cluster (E20)
 ===============================
 
-*Available in Enterprise Edition E20.*
+The high availability cluster feature enables a Mattermost system to maintain service during outages and hardware failures through the use of redundant infrastructure. It is available only in Enterprise Edition E20. 
 
-High Availability support enables a Mattermost server to scale and maintain service during outages and hardware failures through the use of redundant infrastructure. 
+High availability in Mattermost consists of running redundant Mattermost application servers, redundant database servers, and redundant load balancers. The failure of any one of these components does not interrupt operation of the system.
 
-- Support for high availability database configurations is available in Mattermost Enterprise Edition E20 version 3.1 and higher. 
-- Support for high availability configurations of the Mattermost application server is available in Mattermost Enterprise Edition E20 version 3.3 and higher. 
+.. contents::
+  :backlinks: top
+  :local:
+  
+Continuous Operation During Server Updates
+------------------------------------------
 
-This documentation provides a deployment guide, configuration and compatibility details, as well as troubleshooting advice.
+To ensure uninterrupted service during updates, you must make sure that the components are properly sized and that you follow the correct sequence for updating the system components.
 
+Each application server, database, and load balancer must be sized and configured to carry the full load of the system. If this requirement is not met, an outage of one component can result in an overload of the remaining components, causing a complete system outage.
 
-- `Deployment Guide <https://docs.mattermost.com/deployment/cluster.html#id1>`_ - **Instructions to set up and maintain high availability on your Mattermost servers.**
- - `Initial Setup Guide for High Availability <https://docs.mattermost.com/deployment/cluster.html#id2>`_
- - `Adding a Server to the Cluster <https://docs.mattermost.com/deployment/cluster.html#id4>`_
- - `Removing a Server from the Cluster <https://docs.mattermost.com/deployment/cluster.html#id9>`_
+You can apply most configuration changes and dot release security updates without interrupting service, provided that you update the system components in the correct sequence. Exceptions to this rule are changes to configuration settings that require a server restart, and updates that involve a change to the database schema. Downtime for a server restart is around 5 seconds, and for a database schema update, downtime is around 30 seconds.
```

Comment:
Too much text, we need documentation highly **scannable** 

@JeffSchering can we add something to style guide where a) we have "scannable" as a high level objective we're going for, and b) as a guideline, we don't have more than 2-3 sentences without a heading? IT admins today are talking to each other over SnapChat, attention spans aren't as long as they used to be. 

The points we need to make are: 

1. the two conditions needed for continuous operation, and 
2. there's some reasonable exceptions that do require downtime

Proposing we

AVOID BLOCKS OF TEXT: 

The three paragraphs in succession are too difficult to scan: 

![image](https://cloud.githubusercontent.com/assets/177788/22452207/beafcc3a-e729-11e6-8cfd-dad70a84783f.png)

LOOK WE'RE GOING FOR: 

Please try more headings, the following isn't a perfect example, but it has the rhythm: 

![image](https://cloud.githubusercontent.com/assets/177788/22452316/a0ea4198-e72a-11e6-9948-480b1eed160a.png)